src/Symfony/Component/Security/Core/Authorization/TraceableAccessDecisionManager.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+class_alias(TraceableAccessDecisionManager::class, DebugAccessDecisionManager::class);
```

Comment:
I've chosen a different approach: I've removed the `class_exist` (since it seems useless to me) and I've moved it into the deprecated file, which seems the right place to me.